test/hotspot/jtreg/gc/arguments/TestG1CompressedOops.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (C) 2025 THL A29 Limited, a Tencent company. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package gc.arguments;
+
+/*
+ * @test TestG1CompressedOops
+ * @bug 8354145
+ * @requires vm.gc.G1 & vm.opt.G1HeapRegionSize == null
+ * @summary Verify that the flag TestG1CompressedOops is updated properly
+ * @modules java.base/jdk.internal.misc
+ * @modules java.management/sun.management
+ * @library /test/lib
+ * @library /
```

Comment:
Why this line is needed? I don't see any dependencies on "/"
If you use some test code outside directory, better to build them.

Reply:
> Why this line is needed? I don't see any dependencies on "/" If you use some test code outside directory, better to build them.

Yes, the GCArguments depends on the  ```@library /``` , many tests in ``` test/hotspot/jtreg/gc/arguments``` use this

Reply:
Afaict `GCArguments` only depends on `test.lib.*` too.
Other than that, the use of the `@library`directives is often just copy&paste without particular meaning.